script.js

```diff
@@ -0,0 +1,115 @@
+import { filmsSet } from "./data/test-data.js";
+
+const header = {
+  css: "header",
+  paddingX: 20,
+  height: 50,
+  cols: [
+    {
+      view: "label",
+      label: "My App",
+      gravity: 0.2,
+      minWidth: 200
+    },
+    {},
+    {
+      view: "button",
+      type: "icon",
+      icon: "mdi mdi-account",
+      label: "Profile",
+      gravity: 0.2,
+      minWidth: 200
+    }
+  ]
+};
+
+const form = {
+  view: "form",
+  minWidth: 150,
+  maxWidth: 250,
+  elements: [
+    { template: "EDIT FILMS", type: "section" },
+    {
+      margin: 20,
+      cols: [{ view: "label", label: "Title", width: 70 }, { view: "text" }]
+    },
+    {
+      margin: 20,
+      cols: [{ view: "label", label: "Year", width: 70 }, { view: "text" }]
+    },
+    {
+      margin: 20,
+      cols: [{ view: "label", label: "Rating", width: 70 }, { view: "text" }]
+    },
+    {
+      margin: 20,
+      cols: [{ view: "label", label: "Votes", width: 70 }, { view: "text" }]
+    },
+    {
+      margin: 20,
+      cols: [
+        { view: "button", label: "Add new", css: "form-button--add" },
+        { view: "button", label: "Clear" }
+      ]
+    },
+    {}
+  ]
+};
+
+const dataTable = {
+  view: "datatable",
+  minWidth: 540,
+  data: filmsSet,
+  autoConfig: true
+};
+
+const sideMenu = {
```

Comment:
Menu should not fill half page. It should be smaller
![image](https://user-images.githubusercontent.com/47447469/75444483-90e18a80-5974-11ea-9980-3084f31521d7.png)

Reply:
fixed it.